release/gcb/gcb_lib.sh

```diff
@@ -84,9 +84,9 @@ function make_istio() {
   export TAG
   local BRANCH=$5
   ISTIO_OUT=$(make DEBUG=0 where-is-out)
+  ISTIO_OUT_LINUX=$(GOOS=linux make DEBUG=0 where-is-out)
   VERSION="${TAG}"
   export VERSION
-  IFS='/' read -ra REPO <<< "$REL_DOCKER_HUB"
```

Comment:
@utka We changed this so that it is possible to use hubs like 'gcr.io/hubXY'. 
Is there a reason not to use the full REL_DOCKER_HUB (docker.io/istio) vs the truncated REPO (istio)?

Reply:
if we change this should we call this function with `istio`? So the end result doesn't change for our standard releases but it supports other hubs for local ones

Reply:
Yes, we do. In the normal istio . releases, this is called with `docker.io/istio`. So instead of stripping it down to `istio`, we use `docker.io/istio` in the `FROM` statement when adding artifact. But it is locally tagged with this, so it should make no difference, right?

Honestly, we didn't quite understand what this parsing is for. But for a local release this caused some problems. I guess there is a reason, why this is in, but it would be great to understand it.

Reply:
Oh I see, I was thinking this was somehow in the helm values or something but definitely shouldn't be, so this looks good to me. Should double check with utka though since I am also not sure why it's there

Reply:
@utka  I don't quite get it, sorry. 
* For the official release: Whether "docker.io/istio" or "istio" is used does not matter, since the default registry is always "docker.io" (according to e.g. https://github.com/moby/moby/issues/7203)
* For local releases: Using the fully qualified registry name (e.g. "gcr.io/myregistry/istio) is necessary, otherwise the images are pulled from docker.io and not from the private registry/local.

Should `local.sh` work only with 'docker.io' as a registry in your opinion?